pkg/resource/plugin/host.go

```diff
@@ -116,6 +117,7 @@ type defaultHost struct {
 	ctx             *Context                           // the shared context for this host.
 	config          ConfigSource                       // the source for provider configuration parameters.
 	events          Events                             // optional callbacks for plugin load events
+	runtimeOptions  map[string]bool                    // options to pass to the language plugins.
```

Comment:
Are we sure that all options we’ll want to support here are going to be Booleans?

Reply:
I can change this to `interface{}`